web-js/3.x/manual-pan-arrows/README.MD

```diff
@@ -0,0 +1,6 @@
+# Manual Pan Arrows
+
+This sample shows how to programatically pan the map.
```

Comment:
Could you please add a little more detail about how this sample works? You can reference some of the other 3.x/4.x samples for examples.